dbt_subprojects/hourly_spellbook/models/_project/safe/ethereum/safe_ethereum_balances.sql

```diff
@@ -0,0 +1,96 @@
+{{
+    config(
+        schema = 'safe_ethereum',
+        alias = 'balances',
+        partition_by = ['day'],
+        materialized = 'incremental',
+        incremental_strategy = 'merge',
+        file_format = 'delta',
+        unique_key = ['day', 'blockchain', 'address', 'token_address'],
+        on_schema_change = 'sync_all_columns',
```

Comment:
```suggestion
```

Reply:
lets try removing this property next. it's not commonly used across spellbook and just trying to see if this is the issue